cdap-support-bundle/bin/collect_support_bundle.py

```diff
@@ -0,0 +1,561 @@
+            "fileExt": 'log',
+            "fileWrite": False,
+            "forEachNode": True,
+            "placeholderString": "<sts>",
+            "forEachCommand": f'kubectl describe sts <sts> -n {namespace}',
+            "childFileName": "kubectl-describe-sts",
+            "parentFolder": sts_logs
+        },
+        {
+            "command": f'kubectl get deployment -n {namespace}',
+            "fileName": 'kubectl-get-deployment',
+            "fileExt": 'log',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": deployment_logs
+        },
+        {
+            "command": f'kubectl get deployment --no-headers -o custom-columns=":metadata.name" -n {namespace}',
+            "fileName": 'kubectl-get-deployment',
+            "fileExt": 'log',
+            "fileWrite": False,
+            "forEachNode": True,
+            "placeholderString": "<deploy>",
+            "forEachCommand": f'kubectl describe deployment <deploy> -n {namespace}',
+            "childFileName": "kubectl-describe-deployment",
+            "parentFolder": deployment_logs
+        },
+        {
+            "command": f'kubectl get configmap -n {namespace}',
+            "fileName": 'kubectl-get-configmap',
+            "fileExt": 'log',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": config_map_logs
+        },
+        {
+            "command": f'kubectl get configmap --no-headers -o custom-columns=":metadata.name" -n {namespace}',
+            "fileName": 'kubectl-get-configmap',
+            "fileExt": 'log',
+            "fileWrite": False,
+            "forEachNode": True,
+            "placeholderString": "<configmap>",
+            "forEachCommand": f'kubectl describe configmap <configmap> -n {namespace}',
+            "childFileName": "kubectl-describe-configmap",
+            "parentFolder": config_map_logs
+        },
+        {
+            "command": f'kubectl logs -l cdap.container.TetheringAgent -n {namespace}',
+            "fileName": 'kubectl-log-l-cdap-container-TetheringAgent',
+            "fileExt": 'log',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": cdap_services_logs
+        },
+        {
+            "command": f'kubectl logs -l cdap.container.ArtifactCache -n {namespace}',
+            "fileName": 'kubectl-log-l-cdap-container-ArtifactCache',
+            "fileExt": 'log',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": cdap_services_logs
+        },
+        {
+            "command": f'kubectl logs -l cdap.twill.app=system-worker -n {namespace}',
```

Comment:
Let's also pass `--tail -1` arg to ensure that all log lines are shown.

```
$ kubectl logs --help | grep tail
  kubectl logs --tail=20 nginx
      --tail=-1: Lines of recent log file to display. Defaults to -1 with no selector, showing all log lines otherwise 10, if a selector is provided.
```